src/dlgTriggerEditor.cpp

```diff
@@ -7957,6 +7957,18 @@ void dlgTriggerEditor::slot_updateStatusBar(QString statusText)
     QMainWindow::statusBar()->showMessage(stripped);
 }
 
+void dlgTriggerEditor::slot_profileSaveStarted()
```

Comment:
👍  for the visual feedback - do we want an icon change (and/or we can make it a `checkable()` button and do a `setChecked()` at the same time to hold it down and with it being disabled the user cannot click on it again until it is done - though it is important to trigger the save process from the signal that is only sent when the user clicks on the button and not when `setChecked(bool)` is use - that is the difference between `clicked(bool)` and `toggled(bool)` IIRC.)

Might also need to qualify this to avoid unnecessary actions with:

    if (!mProfileSaveAction->checked()) {
        mProfileSaveAction->setChecked(true);
    }

to cover a save active via a different method - lua script command for instance.

Reply:
Probably it is insoluble!

Mind you, just using icons can guarantee the button doesn't change size when the text changes :grin:.  We **could** specify a fixed button size but then we will have a problem when a user/system applied style causes a different font text/size not to fit - and don't forget that this text will be translated and thus can be a significantly different size in other languages (IIRC Chinese and related texts tend to be much shorter as they will only use a few ideographs for a Latinate phrase of several words).

Oh, don't the "Save Profile" and "Save As" buttons have to disable each other as well when one of them is activated?

Reply:
"Save As" works OK for now - and I'd like to move it under "Save Profile" as a submenu eventually

Reply:
👍 It looks better to me that both get disabled now.